config/main.py

```diff
@@ -1555,10 +1555,10 @@ def start(action, restoration_time, ports, detection_time, verbose):
 
     if ports:
         ports = set(ports) - set(['ports', 'detection-time'])
-        cmd += " ports {}".format(' '.join(ports))
+        cmd += " {}".format(' '.join(ports))
 
     if detection_time:
-        cmd += " detection-time {}".format(detection_time)
+        cmd += " {}".format(detection_time)
 
```

Comment:
I was thinking more like:
```
    if action:
        cmd += " --action {}".format(action)

    cmd += " {} {}".format(' '.join(ports), detection_time)

    if restoration_time:
        cmd += " --restoration-time {}".format(restoration_time)

```
But I understand this is a change in command format. If it's not a frequently used command I'd prefer changing the format and use the right logic. If retaining the format is preferred, this change looks fine.